src/legacy/core_plugins/kibana/public/visualize/np_ready/editor/editor.js

```diff
@@ -334,6 +332,24 @@ function VisualizeAppController(
     }
   );
 
+  const updateSavedQueryFromUrl = savedQueryId => {
```

Comment:
This function is only setting `savedQuery`, but it's not triggering the `updateFromSavedQuery` transition (together with syncing the data plugin state) even though it looks like it should.

In 7.6 I can change the saved query in the url and it will re-apply the filters, but here it's just selecting it in the saved object management popover.